tests/manifests/default/config.yaml

```diff
@@ -7,3 +7,6 @@ group=,kind=ConfigMap;group=,kind=ServiceAccount;group=,kind=Secret;group=,kind=
 group=rbac.authorization.k8s.io,kind=RoleBinding;group=rbac.authorization.k8s.io,kind=Role"
 defaultUpgradeStrategy: "pause-and-drain"
 childStatusAssessmentSchedule: "120,60,10"
+permittedRiders:
```

Comment:
can we format this the same as `includedResources` above for similarity? It needs group, not just kind.